aiida/cmdline/utils/echo.py

```diff
@@ -18,7 +18,7 @@
 
 __all__ = (
     'echo', 'echo_info', 'echo_success', 'echo_warning', 'echo_error', 'echo_critical', 'echo_highlight',
-    'echo_dictionary'
+    'echo_dictionary', 'VALID_DICT_FORMATS_MAPPING'
```

Comment:
Why does this need to be exported? I think this should remain internal.

Reply:
Its used here: https://github.com/chrisjsewell/aiida_core/blob/e3b386196eb9d5b4be67b4bf979210269eca1900/aiida/cmdline/params/options/main.py#L581

Reply:
But the fact that it is used internally by the module does not mean it should be in the `__all__`. Even if it is not there, it can still be imported without any issue. So I would really just revert this change as I don't think this should necessarily be exposed at the top level.

Reply:
done 👍